apisix/balancer.lua

```diff
@@ -213,7 +214,9 @@ local function pick_server(route, ctx)
 
     local version = ctx.upstream_version
     local key = ctx.upstream_key
-    local checker = ctx.up_checker
```

Comment:
why change this? I think for single request, we only need fetch health checker once, and reuse same checker during this request by ctx variable.

Reply:
checker is no longer attached to request context and fetched by resource_key and version

Reply:
fixed